multitenancy/tenant-manager-client/src/main/java/io/apicurio/multitenant/client/Auth.java

```diff
@@ -34,6 +34,7 @@ public Auth(String serverUrl, String realm, String clientId, String clientSecret
     }
 
     public String obtainAuthorizationValue() {
+        //FIXME this does not use the refresh token , it requests a new token for every request
         return BEARER + this.keycloak.obtainAccessToken().getToken();
```

Comment:
Same here - we probably need a GH issue for this.

Reply:
Same here.